app/src/main/java/io/seqera/wavelit/App.java

```diff
@@ -79,10 +80,12 @@ public class App implements Runnable {
     @Option(names = {"--context"}, description = "Directory path where the build context is stored.")
     private String contextDir;
 
-
     @Option(names = {"--layer"})
     private List<String> layerDirs;
 
+    @Option(names = {"--config-entrypoint"}, description = "Entrypoint for the container image name to be provisioned.")
+    private List<String> entryPoint;
+
```

Comment:
I think CLI parser expects multiple value to be specified as `--config-entrypoiny foo --config-entrypoint bar` etc, which makes not much sense. 

Also looking to the docker --entrypoint option, it only takes a string value 

```
--entrypoint string              Overwrite the default ENTRYPOINT of the image
```

Reply:
in wave-api, entrypoint is a list
https://github.com/seqeralabs/wave/blob/d15f1c3813dd71a169795e90e987c4a81950de80/wave-api/src/main/java/io/seqera/wave/api/ContainerConfig.java#L18
I will take it here as a string and will pass is as list to wave

Reply:
You can use the `split` option to automatically convert the string to a list based on a separator: https://picocli.info/#_split_regex

Reply:
I think it's enough to convert it to a list of a single item, as done by Docker CLI

Reply:
ok